.github/workflows/main.yml

```diff
@@ -2419,6 +2414,8 @@ jobs:
       matrix:
         java: [ '17' ]
         os: [ 'windows-latest', 'ubuntu-latest' ]
+        exclude:
+          - os: ${{ contains(github.event.pull_request.labels.*.name, 'PHP') || contains(github.event.pull_request.labels.*.name, 'ci:all-tests') || github.event_name != 'pull_request' }} && 'nothing' || 'windows-latest' }}
```

Comment:
this looks a bit complicated but what it does is to only run PHP tests on `ubuntu-latest` if the https://github.com/apache/netbeans/labels/ci%3Aall-tests or https://github.com/apache/netbeans/labels/PHP label is not set. Outside of PRs (e.g on master), or when the label is set, it will run on both win+linux as before.


Why linux? We can run the xserver there + linux nodes are a bit faster.

Reply:
ok this already doesn't work. let me check what is wrong there